week2/homework/src/commands.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable indent */
+const fs = require('fs');
+const chalk = require('chalk');
+let todoList = [];
+
+const list = () => {
+  todoList = JSON.parse(fs.readFileSync('todoList.json'));
+  if (todoList.length === 0) {
+    console.log(`There is nothing to show on your to-do list, please add some tasks`);
+  }
+ else {
+    console.log(todoList);
+  }
+};
+
+const add = todo => {
+  try {
+    todoList = JSON.parse(fs.readFileSync('todoList.json'));
+    console.log(`${todo} is added to your todo list`);
+    todoList.push({ todo });
+
+    fs.writeFileSync('todoList.json', JSON.stringify(todoList));
+  }
+ catch (err) {
+    throw Error(err);

```

Comment:
Throwing an error in a catch may stop your whole program, if it was a server this could be a problem, but sometimes we need, this is fine, but it's not always a good thing to throw an error after catching it, other wise we won't catch:)